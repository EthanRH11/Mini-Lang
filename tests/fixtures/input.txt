
proc inputTest() => {
    int age;
    input<int>("Enter your age: ") => age;

    result => {age};
}


begin:
    
    out_to_console("Your age is: ");
    out_to_console(inputTest());


end