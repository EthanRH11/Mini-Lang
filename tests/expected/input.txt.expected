Your age is: 20